src/openvic-simulation/modifier/StaticModifierCache.cpp

```diff
@@ -1,115 +1,166 @@
 #include "StaticModifierCache.hpp"
-
+#include <string_view>
 #include "openvic-simulation/modifier/ModifierManager.hpp"
+#include "openvic-simulation/dataloader/NodeTools.hpp"
+#include "modifier/Modifier.hpp"
 
 using namespace OpenVic;
+using namespace OpenVic::NodeTools;
+using enum Modifier::modifier_origin_t;
 
 StaticModifierCache::StaticModifierCache()
   : // Country modifiers
-	very_easy_player { nullptr },
-	easy_player { nullptr },
-	hard_player { nullptr },
-	very_hard_player { nullptr },
-	very_easy_ai { nullptr },
-	easy_ai { nullptr },
-	hard_ai { nullptr },
-	very_hard_ai { nullptr },
-	base_modifier { nullptr },
-	war { nullptr },
-	peace { nullptr },
-	disarming { nullptr },
-	war_exhaustion { nullptr },
-	infamy { nullptr },
-	debt_default_to { nullptr },
-	bad_debter { nullptr },
-	great_power { nullptr },
-	secondary_power { nullptr },
-	civilised { nullptr },
-	uncivilised { nullptr },
-	literacy { nullptr },
-	plurality { nullptr },
-	generalised_debt_default { nullptr },
-	total_occupation { nullptr },
-	total_blockaded { nullptr },
+	very_easy_player {"very_easy_player", {}, STATIC},
```

Comment:
```suggestion
	very_easy_player { "very_easy_player", {}, STATIC },
```
Formatting - all of these should have spaces inside the brackets

Reply:
Regex replaced all cases